### ISSUE TITLE
Invalid argument: unsupported datatype 'TYPE_BYTES' on 20.08

### ISSUE THREAD

**Issue by Windfarer** [triton-inference-server/server#1988]
**Description**
Using `nvcr.io/nvidia/tritonserver:20.08-py3` to load a Tensorflow 1.15 savedmodel, and get the error log as follows 
```
triton_1  | E0904 09:38:33.860808 1 model_repository_manager.cc:928] failed to load 'test_model' version 1: Invalid argument: unsupported datatype 'TYPE_BYTES' for tensor 'feature1' for model 'test_model'
```

but the same model can be successful loaded with `nvcr.io/nvidia/tritonserver:20.07-py3`

I also try set `--backend-config=tensorflow,version=1` to the new version, but still shows that error.

In addition, I can make sure this tensor in my model is DT_STRING type, not the BYTES type.

It seems this is a compatible issue between tf 1.x and 2.x

**Triton Information**
Release 2.2.0 corresponding to NGC container 20.08

Are you using the Triton container or did you build it yourself?
use official image nvcr.io/nvidia/tritonserver:20.08-py3

**To Reproduce**
maybe load a tf 1.x model with DT_STRING tensor will reproduce this error

**Expected behavior**
 tf 1.15 savedmoel can be loaded the same as old version.

**Comment by CoderHam** [comment]
@Windfarer can you share your model?

**Comment by CoderHam** [comment]
Reproduced and triaged. Could you please try the changes above with your model to confirm?

**Comment by Windfarer** [comment]
The image built from fixed code is able to load my model successfully. Excellent !

**Comment by fsatka** [comment]
I have same problem with docker image `nvcr.io/nvidia/tritonserver:20.08-py3`.
Can you update it please?

**Comment by deadeyegoodwin** [comment]
This issue is fixed in the 20.09 release, which is releasing later this week.
